BlueprintUICommonControls/Sources/PointerInteraction.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  PointerInteraction.swift
+//  BlueprintUICommonControls
+//
+//  Created by Kyle Van Essen on 6/26/20.
+//
+
+import UIKit
+import BlueprintUI
+
+
+
+/// An `Element` which lets you add `UIPointerInteraction` interactions
+/// to the contained `Element`.
+///
+/// A `UIPointerInteraction` allows you to respond to hover events for buttons,
+/// text, icons, etc, when a user is using an iPad that has support for a pointer, usually from a trackpad.
+///
+/// **Note** – On iOS versions before 13.4, this has no effect.
+///
+public struct PointerInteraction : Element
+{
+    /// The element to which the pointer action will be applied.
+    public var wrapping : Element
+    
+    /// How to provide the style based on the coordinate space and pressed keys.
+    public typealias StyleProvider = (UICoordinateSpace, UIKeyModifierFlags) -> Style
+    
+    /// How to provide the style based on the coordinate space and pressed keys.
+    public var style : StyleProvider
+    
+    /// Creates a new `PointerInteraction` instance that wraps the provided element,
+    /// styling the interaction via the `StyleProvider`.
+    ///
+    /// You don't need to provide a `StyleProvider` – the default one will simply
+    /// return a `.automatic` style.
+    ///
+    /// **Note** – On iOS versions before 13.4, this has no effect.
```

Comment:
If you do ` - Note: Foo` it makes a header in the rendered docs. Just fyi - no need to update unless you want to.

<img width="216" alt="Screen Shot 2020-10-27 at 2 01 16 PM" src="https://user-images.githubusercontent.com/3526783/97361593-fd42bf80-185c-11eb-99c9-056f6133cdcd.png">

Reply:
(**Discussion** is automatic, for any docs after a newline)

Reply:
Oh yeah... I go back and forth on this, since the `- Note: Foo` is more immediately obvious in the generated docs, but the `**note**` when reading the code directly is more immediately obvious. I can convert it though, since I bet `- **Note**: Foo` works too.